haros/haros.py

```diff
@@ -317,10 +319,14 @@ def main(argv = None, source_runner = False):
                             level = logging.DEBUG)
     else:
         logging.basicConfig(level = logging.WARNING)
+
     try:
+        if args.dir:
+            os.chdir(args.dir)
```

Comment:
Just to wrap up this thread, ROS2 is in the to-do list, so yes, Windows support will eventually be a concern. I actually work on a Windows machine most of the time, and I can tell you that `chdir` does behave the same as in Unix. While redundant, it does not really hurt either.